src/pages/blog.tsx

```diff
@@ -85,6 +83,14 @@ const BlogPage: React.FC = () => {
         <GridItem>
           <BlogPageTitle>Blog</BlogPageTitle>
         </GridItem>
+        <GridItem md={8}>
+          <BlogPageSubTitl>
+            In our blog, our developers and designers talk about the latest
+            trends and know-how around tech and especially about our latest
+            learnings and insights on how to create incredibly good software for
+            the web.
```

Comment:
> create incredibly good software for the web.
might be true but that sounds incredible over the edge 😁

What about:
> Our developers and designers talk about the latest trends and know-how around tech, especially about our latest learnings and insights on creating good software for the web.

Reply:
Took the text from the design, but yes, maybe a bit too much :D Updated it to your suggestion